### PR TITLE
Remove GUI

### PR DESCRIPTION
**Pull request by Halbaroth**
This PR removes completely the old GUI.

### CODE REVIEW

**Comment by Halbaroth** [comment]
It is ready for review but please do not merge it.

**Comment by Halbaroth** [comment]
There is no complaint about this PR. We should review it now :)

**Review by bclement-ocp — APPROVED** [review]
:+1:

**Pull request merged**
